clients/src/main/java/org/apache/kafka/common/config/internals/BrokerSecurityConfigs.java

```diff
@@ -67,4 +68,9 @@
             + "Only GSSAPI is enabled by default.";
     public static final List<String> DEFAULT_SASL_ENABLED_MECHANISMS = Collections.singletonList(SaslConfigs.GSSAPI_MECHANISM);
 
+    public static final String SASL_SERVER_CALLBACK_HANDLER_CLASS_MAP_DOC = "A map between SASL mechanisms and SASL server " +
+            "callback handler classes that implement the AuthenticateCallbackHandler interface.  Key and value are " +
+            "separated by a colon and map entries are separated by commas. For example, " +
+            "'PLAIN=CustomPlainCallbackHandler,SCRAM-SHA-256=CustomScramCallbackHandler'.";
```

Comment:
Example should use ":" instead of "=": PLAIN:CustomPlainCallbackHandler,SCRAM-SHA-256:CustomScramCallbackHandler